dist/test_data.py

```diff
@@ -218,15 +218,15 @@ def __ge__(self, other):
 ]
 
 methods = {
-    'base_test' : Method(test_config),
```

Comment:
Can you mention in the JIRA/commit that we're also renaming these tests?

Reply:
It is present in the JIRA ticket!